src/bitcoin-cli.cpp

```diff
@@ -291,6 +293,184 @@ class GetinfoRequestHandler: public BaseRequestHandler
     }
 };
 
+/** Process netinfo requests */
+class NetinfoRequestHandler : public BaseRequestHandler
+{
+private:
+    bool IsAddrIPv6(const std::string& addr) const
+    {
+        return !addr.empty() && addr.front() == '[';
+    }
+    bool IsInboundOnion(int mapped_as, const std::string& addr, const std::string& addr_local) const
+    {
+        return mapped_as == 0 && addr.find("127.0.0.1") == 0 && addr_local.find(".onion") != std::string::npos;
+    }
+    bool IsOutboundOnion(const std::string& addr) const
+    {
+        return addr.find(".onion") != std::string::npos;
+    }
+    bool m_verbose{false}; //!< Whether user requested verbose -netinfo report
+
+    enum struct m_conn_type {
```

Comment:
This and `m_peer` are types, not member variables. I think they should be named `ConnType` and `Peer`.

Reply:
I hesitated on this. Will update in a follow-up.